packages/permissions/src/components/authorized/authorized.tsx

```diff
@@ -95,6 +95,7 @@ const injectAuthorized =
           options.getSelectDataFenceData(props)
         }
         render={(isAuthorized) => (
+          // @ts-ignore Related to https://github.com/emotion-js/emotion/issues/3245
```

Comment:
With the latest changes in `main`, perhaps this ignore directive is not needed anymore. Could you please double check? 🙏